bootstrap/bootstrap/src/config.rs

```diff
@@ -119,12 +119,15 @@ impl Config {
         if stage == 0 {
             bail!("invalid stage value 0, stages starting from 1");
         }
-
+        let mut ver = Version::V3;
```

Comment:
So, looks like it will become the version latest-1?

what happens when we upgrade  `zos`  with `latest - 2` or `latest` version?
In this context, it means v2 or v4

Reply:
at 1st step we set it to v3 this is the current supported version so it will not fail at 1st step